File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ShikimoriSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await ShikimoriSDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# Shikimori SDK exists test

require "minitest/autorun"
require_relative "../Shikimori_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = ShikimoriSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
